src/Mvc/Mvc.ApiExplorer/src/EndpointMetadataApiDescriptionProvider.cs

```diff
@@ -222,29 +216,13 @@ private ApiDescription CreateApiDescription(RouteEndpoint routeEndpoint, string
             }
         }
 
-        private static void AddSupportedRequestFormats(
-            IList<ApiRequestFormat> supportedRequestFormats,
-            bool hasJsonBody,
-            EndpointMetadataCollection endpointMetadata)
+        private static void AddSupportedRequestFormats(IList<ApiRequestFormat> supportedRequestFormats, IAcceptsMetadata acceptsMetadata)
```

Comment:
Do you even need this method still?

Reply:
yes, for adding the user specified media-types. Are you suggesting we move the the logic inside the ```CreateApiDescription``` ?

Reply:
Is it called from more than one place?

Reply:
yes, a single place.

Reply:
Then remove the function and just replace it with the code. It's become a trivial function now and there's no need for it.